README.md

```diff
@@ -14,12 +14,27 @@ This plugin is written in Groovy, and therefore needs to have Groovy installed o
 
 For a full list of tasks, see the output of:
 
-```bash 
+```bash
 ./gradlew tasks
 ```
 
 Note that you will not be able to publish to plugins.gradle.org. To get a new version uploaded, contact 333fred.
 
+To publish for local testing, use the following command.
+```bash
+./gradlew publishToMavenLocal -PlocalPublish
```

Comment:
Rather than this, is it possible to detect that the publishToMavenLocal task was the one that was called?

Reply:
And by this, I mean the `-PlocalPublish`. I know I've done task detection in the past, but I can't remember how exactly.

Reply:
I'll take another look. When I was writing this last year by the time I could detect publishToMavenLocal was running it wouldn't let me set those properties. But maybe I missed something.

Reply:
Both `gradle.taskGraph.whenReady {taskGraph ->` and `tasks.whenTaskAdded { task ->` happen too late. Its not possible to modify the groupId after the initial creation closure. It doesn't seem to be on any publicly exposed property after the fact.

Reply:
Alright then, guess we have to go with the property.

Reply:
If I ever find another way to do it, We use it in like 5 repos so I'll definitely update this one too. I'll ask in gradles slack support as well.